.tekton/rhtap-consoledot-frontend-build-tests-pull-request.yaml

```diff
@@ -208,6 +210,21 @@ spec:
       workspaces:
       - name: source
         workspace: workspace
+    - name: inject-dockerfile
+      taskRef:
+        resolver: git
+        params:
+          - name: git-url
+            value: $(params.git-url)
+          - name: revision
+            value: main
```

Comment:
and this one should be $(params.revision) because we want to use the same revision as the pipeline